lib/js/app/queryCreator/components/QueryArguments/QueryArguments.tsx

```diff
@@ -34,10 +37,11 @@ const App: FC<Props> = () => {
   const timeframe = useSelector(getTimeframe);
   const targetProperty = useSelector(getTargetProperty);
   const timezone = useSelector(getTimezone);
+  const percentile = useSelector(getPercentile);
 
   return (
     <Container>
-      <MenuItem>
+      <MenuItem width="25%">
```

Comment:
Please, correct me if I am wrong but Is defining a percentage width for each `<MenuItem>` component is necessary ? From my point of view flex should adjust all items automatically and the Percentile component could have static width.